corehq/util/view_utils.py

```diff
@@ -69,7 +70,7 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
-            notify_exception(request, 'JSON exception response: {}'.format(e))
+            notify_exception(request, 'JSON exception response: {}'.format(smart_bytes(e)))
```

Comment:
lol

On Thu, Jul 7, 2016 at 9:11 AM, Norman Hooper <notifications@github.com>
wrote:

> In corehq/util/view_utils.py
> <https://github.com/dimagi/commcare-hq/pull/12224#discussion_r69902908>:
>
> > @@ -69,7 +70,7 @@ def inner(request, *args, **kwargs):
> >          except BadRequest as e:
> >              return _get_json_exception_response(400, request, e)
> >          except Exception as e:
> > -            notify_exception(request, 'JSON exception response: {}'.format(e))
> > +            notify_exception(request, 'JSON exception response: {}'.format(smart_bytes(e)))
>
> Agreed. I feel this is a pragmatic solution given our situation. (Is that
> a Pink Floyd reference?! "Hey you, standing in the ... aisles"? ... Don't
> help them to bury the light, @millerdev <https://github.com/millerdev>.
> Don't give in without a fight! 😆)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/12224/files/a5e9939661a7c3e8d4b4f667ac9a1c859a7c357f..415249fc567656d3333bd9910ea3843e461d6796#r69902908>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA4D8toaWSEYFr72-clTFvGTfpsolCloks5qTPsRgaJpZM4I9zhs>
> .
>